core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -658,19 +661,16 @@ public StartupConsistencyCheck getStartupConsistencyCheck() {
    * @throws AccessControlException if permission checking fails
    */
   public long getFileId(AlluxioURI path) throws AccessControlException {
-    long flushCounter = AsyncJournalWriter.INVALID_FLUSH_COUNTER;
-    try (LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.WRITE)) {
+    try (JournalContext journalContext = createJournalContext();
```

Comment:
I don't know if I missed it, but could you add a comment to the top of the file to define this convention? In try resource order, the journal context must be first, so that it is closed last.